mbuild/compound.py

```diff
@@ -1154,20 +1156,71 @@ def center(self):
             return np.mean(self.xyz, axis=0)
 
     @property
-    def boundingbox(self):
+    def mins(self):
+        """Return the mimimum x, y, z coordinate of any particle in this compound."""
+        return self.xyz.min(axis=0)
+
+    @property
+    def maxs(self):
+        """Return the maximum x, y, z coordinate of any particle in this compound."""
+        return self.xyz.max(axis=0)
+
+    def get_boundingbox(self):
         """Compute the bounding box of the compound.
 
+        Compute and store the rectangular bounding box of the Compound.
+
         Returns
         -------
         mb.Box
-            The bounding box for this Compound
+            The bounding box for this Compound.
+
+        NOTE
+        ----
```

Comment:
```suggestion
        Notes
        -----
```
Make rtd happy https://numpydoc.readthedocs.io/en/latest/format.html